src/components/BadgerButton/stories.js

```diff
@@ -289,14 +289,56 @@ storiesOf('BadgerButton', module)
 		}
 	)
 	.add(
-		'BIP070 Invoicing',
```

Comment:
match previous storybook naming convention for BIP070 examples

Reply:
102bd63